hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/AvailabilityConditionEval.java

```diff
@@ -31,6 +31,8 @@
  */
 public class AvailabilityConditionEval extends ConditionEval {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
I thought that serialVersionUID variable from implementing Serializable are not mandatory with newer versions of JDK if I am not wrong.